std_msgs/Header header
#int64 thrust_ref      # thrust_ref reading (from -999 to +200)

float64 thrust_ref      # thrust_ref reading (from -999 to +200)
